### ISSUE TITLE
WordPress importer needs a “change URL” function, along with more docs

### ISSUE THREAD

**Issue by asmeurer** [getnikola/nikola#1094]
#### TL;DR:

We should:
- [ ] provide an option for overriding the URL from the WordPress dump
- [ ] provide another option that would basically go crazy and run `sed s/old_url/new_url/` on all the imported posts.  This should be optional, for safety.
- [ ] Add some warnings on “what is really important to change” to imports to existing sites.
- [ ] document the WordPress importer properly
#### Original post follows.

I have had quite a few issues with the Wordpress importer. You can see everything I've done at https://github.com/asmeurer/blog (and some old work at https://github.com/asmeurer/asmeurer.github.com). I have the wordpress import file there too. Some issues:
- Apparently the conf.py generated by the importer is wrong. It lists the site url as the wordpress site.
- I was a little confused by the whole process. Why do you have to import into a new blog? It would be much easier to import into an existing blog.
- The comment translation csv was not very helpful. This was the map it gave me https://github.com/asmeurer/blog/blob/d9b32caacc51c7b93f10895b0e0ac541c000daa0/old_blog/url_map.csv. Unless I am still misunderstand this (which I may be, since I haven't gotten it to work yet), that is incorrect. 

In general, at no point did the importer actually ask me for the new site url, which was a bit confusing, since it seems to me that it should have it for several things.

**Comment by fuzzy76** [comment]
Is this still the correct way of getting Disqus working? 

1. Import the file you got from WordPress at Disqus.
2. Get the CSV file for URL mapping.
3. Create the URL map by hand.

If so, where do I get the CSV file? And what is this URL map? Is its existence (and purpose) documented anywhere? What do I do with it?

**Comment by Kwpolska** [comment]
The file is available on Disqus (go to import → from CSV file). You need to tell Disqus that all your URLs were changed by migrating from WordPress to Nikola.

(You don’t need to do this if you want to get rid of all old comments, of course)

**Comment by fuzzy76** [comment]
Ok, so I got the file. What is this URL map, and where does it go? Do I have to manually edit the URL of each post to match the old ones?

**Comment by Kwpolska** [comment]
you need to create a list in format:

    oldURL,newURL

Nikola generates an url_map.csv file, that might help you do it automatically (but it’s not ready for Disqus import). There’s no way to do it fully automated (yet).
